sdcm/cluster.py

```diff
@@ -734,15 +736,18 @@ def _get_coredump_backtraces(self, last=True):
         :return: fabric.Result output
         """
         try:
-            backtrace_cmd = 'sudo coredumpctl info'
+            backtrace_cmd = 'sudo coredumpctl info --no-pager --no-legend'
             if last:
                 backtrace_cmd += ' -1'
             return self.remoter.run(backtrace_cmd,
                                     verbose=False, ignore_status=True)
+        except NETWORK_EXCEPTIONS:  # pylint: disable=W0706
```

Comment:
in pylint disables please use the names, and not the numbers

Reply:
fixed